### PR TITLE
PIMOB-1359: Visa card should allow 19-digit length

### PR DESCRIPTION
**Pull request by alex-ioja-yang-cko**
## Proposed changes

[Jira task](https://checkout.atlassian.net/browse/PIMOB-1359)

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [X] Bugfix (non-breaking change which fixes an issue)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [X] Lint and unit tests pass locally with my changes
* [X] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

Small change to enable VISA cards to input up to 19 characters

### CODE REVIEW

File: Checkout/Checkout/Source/Model/Card+Scheme.swift [hunk 1]
```diff
@@ -138,8 +138,9 @@ extension Card {
 
     var maxCardLength: Int {
       switch self {
+      case .visa:
+          return 19
```
Comment (ehab-al-cko) [inline Checkout/Checkout/Source/Model/Card+Scheme.swift:142]: i am wondering about this logic, it should allow 19 or 16 or 19 only?

Comment (alex-ioja-yang-cko) [reply Checkout/Checkout/Source/Model/Card+Scheme.swift:142]: Most Visa cards are 16 digits. Apparently they can also be 19 characters long. So both 16 and 19 are valid, and the regex seems to already support 16 and 19. The only issue was with allowing input to reach 19.

Comment (patrick-hoban-cko) [reply Checkout/Checkout/Source/Model/Card+Scheme.swift:142]: Yep - looks okay, as that variable sets the _maxCardLength_, which in VISA's case, is indeed 19.

**Review by deepesh-vasthimal-cko — APPROVED** [review]

**Comment by ehab-al-cko** [comment]
@alex-ioja-yang-cko  
Kind reminder, please change `alex/` to `bugfix/` or `feature/` next time. we are following `git flow`

**Review by ehab-al-cko — APPROVED** [review]

**Comment by ehab-al-cko** [comment]
@alex-ioja-yang-cko  On merge please change commit to 
`Merge pull request #234 from checkout/bugfix/PIMOB-1359-i-os-checkout-i-os-card-validator-should-allow-visa-cards-of-19-digit-length`

**Pull request merged**